packages/hub/node-tests/@core/babel-plugin-card-schema-transform-test.ts

```diff
@@ -333,7 +333,7 @@ if (process.env.COMPILER) {
       let source = getFileCache().getModule(compiled.schemaModule.global, 'browser');
       expect(source).to.containsSource(`
         static loadedFields(schemaInstance) {
-          return [...schemaInstance.loadedFields];
+          return allFields.length === 0 ? [] : [...schemaInstance.loadedFields];
```

Comment:
what is `schemaInstance.loadedFields` in this case? ideally it should be an empty array.